src/pages/Main/index.jsx

```diff
@@ -12,8 +20,14 @@ class Main extends React.Component {
     render() {
         return (
             <Fragment>
-                <div>Main</div>
-                <NavLink to={'../'}>dupa</NavLink>
+                <Header />
+                <Switch>
+                    <Route exact path={`${this.props.match.path}`} component={Dashboard} />
+                    <Route path={`${this.props.match.path}/profile`} component={Profile} />
+                    <Route path={`${this.props.match.path}/calendar`} component={Calendar} />
```

Comment:
since it is repeated 3 times, what do you think about destructuring `path` from `this.props.match`?

`const { path } = this.props.match`

Reply:
sure :)